go.mod

```diff
@@ -138,9 +138,14 @@ require (
 )
 
 require (
+	github.com/agext/levenshtein v1.2.1 // indirect
+	github.com/apparentlymart/go-textseg/v15 v15.0.0 // indirect
 	github.com/go-ini/ini v1.67.0 // indirect
 	github.com/google/jsonschema-go v0.3.0 // indirect
+	github.com/hashicorp/hcl/v2 v2.24.0 // indirect
```

Comment:
https://github.com/lima-vm/lima/actions/runs/20217415615/job/58032762244?pr=4460

> Not allowed license MPL-2.0 found for library github.com/hashicorp/hcl/v2

Reply:
Likely covered in:
https://github.com/cncf/foundation/blob/e805aef48792cbbab4b8aa7b872999e44ada37aa/license-exceptions/CNCF-licensing-exceptions.csv#L427